contracts/contracts/TACoApplication.sol

```diff
@@ -618,16 +625,14 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
     /**
      * @notice Returns staking provider for specified operator
      */
-    function stakingProviderFromOperator(address _operator) external view returns (address) {
+    function operatorToStakingProvider(address _operator) external view returns (address) {
```

Comment:
This will require changes to nucypher. Opened https://github.com/nucypher/nucypher/issues/3350